src/Terrain/PlanetWrenderer.h

```diff
@@ -1,46 +1,28 @@
 // "PlanetRenderer is a little too boring" -- Capital Asterisk, 2018
 #pragma once
 
-#include <cstdint>
```

Comment:
Removed a bunch of header files that weren't needed for this header.

I did confirm that everything still compiles.